src/Collections/MemoryConfigCollection.php

```diff
@@ -143,10 +137,7 @@ protected function getClassConfig($class, $excludeMiddleware = 0)
             $class,
             $excludeMiddleware,
             function ($class, $excludeMiddleware) {
-                $classKey = strtolower($class);
-                return isset($this->config[$classKey])
-                    ? $this->config[$classKey]
-                    : [];
+                return $this->getClassConfig($class, true);
```

Comment:
could this turn recursive? it almost seems that way if you're not careful about the `callMiddleware`

Reply:
No because the "true" disables middleware.

Reply:
oh, I see now :)